src/doom/d_main.c

```diff
@@ -834,11 +841,19 @@ void D_IdentifyVersion(void)
     }
 }
 
+
+static void D_FreeGameDescription(void)
+{
+    free(gamedescription);
+    gamedescription = NULL;
+}
+
+
 // Set the gamedescription string
 
-void D_SetGameDescription(void)
+static void D_SetGameDescription(void)
 {
-    gamedescription = "Unknown";
+    I_AtExit(D_FreeGameDescription, false);
```

Comment:
Please don't do this.

https://devblogs.microsoft.com/oldnewthing/20120105-00/?p=8683

Reply:
I know that all memory gets freed at exit, however I still find it useful to clean up everything because it increases the usefulness of leak detectors (Valgrind/AddressSanitizer). When there's a bunch of leaks which are expected to be cleaned up automatically it can hide actual leaks. But if there are *no* leaks then I can be sure there are no unbounded leaks either and I'm willing to pay a few milliseconds at exit for knowing this.

But since you don't want it I've taken it out. I'll put it up on my repo later (and other leak fixes) in case anyone else wants it.